sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/MessageScheduleAsyncSample.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Sample demonstrates how to schedule {@link ServiceBusMessage} to an Azure Service Bus queue or topic.
+ */
+public class MessageScheduleAsyncSample {
+
+    /**
+     * Main method to invoke this demo on how to send a message to an Azure Service Bus.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // The connection string value can be obtained by:
+        // 1. Going to your Service Bus namespace in Azure Portal.
+        // 2. Go to "Shared access policies"
+        // 3. Copy the connection string for the "RootManageSharedAccessKey" policy.
+        String connectionString = System.getenv("AZURE_SERVICEBUS_CONNECTION_STRING");
+
+        // Instantiate a client that will be used to call the service.
+        ServiceBusSenderAsyncClient senderAsyncClient = new ServiceBusClientBuilder()
+            .connectionString(connectionString)
+            .sender()
+            .queueName("<< QUEUE NAME >>")
+            .buildAsyncClient();
+
+        // Create an message to send.
+        ServiceBusMessage message = new ServiceBusMessage("Hello World!!".getBytes(UTF_8));
+
+        // Send that message. This call returns a Mono<Void>, which we subscribe to. It completes successfully when the
+        // message has been delivered to the Service Bus. It completes with an error if an exception occurred while sending
+        // the message.
+
+        senderAsyncClient.scheduleMessage(message, Instant.now().plusSeconds(1 * 60L))
```

Comment:
Why not .plusSeconds(60) or plusMinute(1)?

Reply:
I did not see plusMinute() API .